all_age_brackets_template.json

```diff
@@ -0,0 +1,92 @@
+{ 
+    "County Name": {
+        "data": {
+            "cases": {
+                "age_group_1": { // Age Brackets for SF, Santa Clara, Alameda
+                    "30 or younger": -1,
+                    "31-40": -1,
+                    "41-50": -1,
+                    "51-60": -1,
+                    "61-70": -1,
+                    "71-80": -1,
+                    "81-90" : -1,
+                    "90+": -1,
+                    "Unknown": -1
+                },
+                "age_group_2": { // Marin, Solano
+                    "0-18": -1,
+                    "19-64": -1,
+                    "65+":
+                    
+                },
+                "age_group_3": { // Sonoma, Napa
+                    "0-17": -1,
+                    "18-49": -1,
+                    "50-64": -1,
+                    "64+": -1,
+                    "Unknown": -1
+                },
+                "age_group_4": { // San Mateo
+                    "0 to 19": -1,
+                    "20-29": -1,
+                    "30-39": -1,
+                    "40-49": -1,
+                    "50-59": -1,
+                    "60-69": -1,
+                    "70-79": -1,
+                    "80-89": -1,
+                    "90+": -1
+                    
+                },
+
+                "age_group_5": { // Contra Costa
+                    "0-20": -1,
+                    "21-40": -1,
+                    "41-60": -1,
+                    "61-80": -1,
+                    "81-100": -1,
+                }
+
+            }, 
+            "deaths": { // death by age data only available for SF, Santa Clara, San Mateo. Marin
+                "age_group_1": { // Age Brackets for SF
+                    "Under 60": 1,
+                    "60+": 4
+                },
+                "age_group_2": { // Age Brackets for Santa Clara
+                    "30 or younger": -1,
+                    "31-40": -1,
+                    "41-50": -1,
+                    "51-60": -1,
+                    "61-70": -1,
+                    "71-80": -1,
+                    "81-90" : -1,
+                    "90+": -1,
+                    "Unknown": -1
+                },
+
+                "age_group_3": { // Marin,
+                    "0-18": -1,
+                    "19-64": -1,
+                    "65+": -1
+                    
+                },
```

Comment:
Earlier in the weekend it seemed that we wanted to merge the age brackets when possible, but just read your comment above - "If each county provides different age brackets, the easiest way around this may be to simply use each county's provided age brackets rather than try to normalize them. I'm worried that normalizing them will lose a lot of the granularity that makes the age data interesting. I think that the charting library should be able to handle different brackets. We can make a note of why the brackets differ from county to county.". 

If I'm understanding things correctly, we can just ignore this age brackets json since we won't be merging age brackets. That's fine with me!

Reply:
Yeah, the more I think about this, I think it makes more sense to not merge the age groups